package.json

```diff
@@ -21,89 +21,89 @@
+    "@connectrpc/connect": "^1.6.1",
+    "@connectrpc/connect-web": "^1.6.1",
+    "@penumbra-labs/registry": "^11.4.0",
+    "@penumbra-zone/bech32m": "^10.0.0",
+    "@penumbra-zone/client": "^21.0.0",
+    "@penumbra-zone/getters": "^20.0.0",
+    "@penumbra-zone/protobuf": "^6.3.0",
     "@penumbra-zone/transport-dom": "^7.5.0",
-    "@penumbra-zone/types": "^25.0.0",
-    "@penumbra-zone/ui": "^11.0.2",
-    "@penumbra-zone/wasm": "^30.1.0",
+    "@penumbra-zone/types": "^26.0.0",
+    "@penumbra-zone/wasm": "^31.0.0",
+    "@radix-ui/react-avatar": "^1.1.1",
+    "@radix-ui/react-checkbox": "^1.1.2",
+    "@radix-ui/react-dialog": "1.0.5",
+    "@radix-ui/react-dropdown-menu": "^2.1.2",
     "@radix-ui/react-icons": "^1.3.0",
-    "@rehooks/component-size": "^1.0.3",
-    "@styled-icons/octicons": "^10.47.0",
-    "@tanstack/react-query": "^5.59.0",
+    "@radix-ui/react-navigation-menu": "^1.2.1",
+    "@radix-ui/react-popover": "^1.1.2",
+    "@radix-ui/react-progress": "^1.1.0",
+    "@radix-ui/react-radio-group": "^1.2.1",
+    "@radix-ui/react-select": "^2.1.2",
+    "@radix-ui/react-slider": "^1.2.1",
+    "@radix-ui/react-slot": "^1.1.0",
+    "@radix-ui/react-switch": "^1.1.1",
+    "@radix-ui/react-tabs": "^1.1.1",
+    "@radix-ui/react-toast": "^1.2.2",
+    "@radix-ui/react-toggle": "^1.1.0",
+    "@radix-ui/react-tooltip": "^1.1.3",
+    "@tanstack/react-query": "^5.59.16",
     "@tsconfig/strictest": "^2.0.5",
     "@tsconfig/vite-react": "^3.0.2",
-    "@vercel/analytics": "^1.3.1",
-    "@vercel/speed-insights": "^1.0.12",
+    "@vercel/analytics": "^1.3.2",
+    "@vercel/speed-insights": "^1.0.14",
     "bech32": "^2.0.0",
     "bignumber.js": "^9.1.2",
-    "chart.js": "^4.4.4",
-    "chartjs-adapter-date-fns": "^3.0.0",
-    "chartjs-chart-financial": "^0.2.1",
-    "chartjs-plugin-annotation": "^3.0.1",
-    "chartjs-plugin-zoom": "^2.0.1",
-    "configs": "https://github.com/prax-wallet/configs.git#main",
+    "configs": "github:prax-wallet/configs#main",
     "date-fns": "^3.6.0",
-    "echarts": "^5.5.1",
-    "echarts-for-react": "^3.0.2",
-    "framer-motion": "^11.3.31",
-    "grpc-tools": "^1.12.4",
-    "grpc_tools_node_protoc_ts": "^5.3.3",
-    "lightweight-charts": "^4.2.0",
+    "framer-motion": "^11.11.10",
+    "lightweight-charts": "^4.2.1",
     "lodash": "^4.17.21",
-    "lucide-react": "^0.445.0",
-    "mobx": "^6.13.3",
+    "lucide-react": "^0.453.0",
+    "mobx": "^6.13.5",
     "mobx-react-lite": "^4.0.7",
-    "next": "^14.2.7",
-    "pg": "^8.12.0",
+    "murmurhash3js": "^3.0.1",
+    "next": "^14.2.16",
+    "pg": "^8.13.1",
     "prettier": "^3.3.3",
     "react": "^18.3.1",
-    "react-chartjs-2": "^5.2.0",
     "react-dom": "^18.3.1",
-    "react-outside-click-handler": "^1.3.0",
-    "sharp": "^0.33.5",
-    "webpack": "^5.94.0",
+    "tinycolor2": "^1.6.0",
+    "webpack": "^5.95.0",
     "zod": "^3.23.8"
   },
   "devDependencies": {
-    "@babel/core": "^7.25.2",
```

Comment:
- removed legacy dependencies
- updated existing dependencies
- added many libraries needed to support the UI package